boskos/crds/client.go

```diff
@@ -111,233 +105,50 @@ type Collection interface {
 	GetItems() []Object
 }
 
-// createRESTConfig for cluster API server, pass empty config file for in-cluster
-func createRESTConfig(kubeconfig string, t Type) (config *rest.Config, types *runtime.Scheme, err error) {
-	if kubeconfig == "" {
-		config, err = rest.InClusterConfig()
-	} else {
-		config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
-	}
-
-	if err != nil {
-		return
-	}
-
-	version := schema.GroupVersion{
-		Group:   group,
-		Version: version,
-	}
-
-	config.GroupVersion = &version
-	config.APIPath = "/apis"
-	config.ContentType = runtime.ContentTypeJSON
-
-	types = runtime.NewScheme()
-	schemeBuilder := runtime.NewSchemeBuilder(
-		func(scheme *runtime.Scheme) error {
-			scheme.AddKnownTypes(version, t.Object, t.Collection)
-			v1.AddToGroupVersion(scheme, version)
-			return nil
-		})
-	err = schemeBuilder.AddToScheme(types)
-	config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: serializer.NewCodecFactory(types)}
-
-	return
-}
-
-// registerResource sends a request to create CRDs and waits for them to initialize
-func registerResource(config *rest.Config, t Type) error {
+// registerResources sends a request to create CRDs
+func registerResources(config *rest.Config) error {
 	c, err := apiextensionsclient.NewForConfig(config)
 	if err != nil {
 		return err
 	}
 
-	crd := &apiextensionsv1beta1.CustomResourceDefinition{
```

Comment:
we could've kept this generic and looped over the two types, right?

I guess the new version is a little easier to read.

Reply:
Well yeah, looping would require to have a slice somewhere everyone who could think about adding a CRD will definitely find and I was too lazy to think about the best place for that :)

IMHO we should drop the whole CRD creation entirely, as that requires us to give CRD Create permissions, which shouldn't be needed. I didn't do that in this PR to keep the potential basis for debate as minimal as possible :)

Reply:
> IMHO we should drop the whole CRD creation entirely, as that requires us to give CRD Create permissions, which shouldn't be needed. I didn't do that in this PR to keep the potential basis for debate as minimal as possible :)

I agree. https://github.com/kubernetes/test-infra/pull/16266